app/components/admin/statements/filter.html.erb

```diff
@@ -0,0 +1,45 @@
+<%= form_with url: admin_finance_statements_path, method: :get do |f| %>
+  <div class="admin-statements-filter">
+      <div class="govuk-form-group filter-form-group">
```

Comment:
Indentation here jumps from 2 to 4 spaces.

Reply:
removed spacing